CONTRIBUTING.rst

```diff
@@ -47,6 +51,17 @@ for committers:
   merged until this is rectified.
 * All merged patches must have 100% test coverage.
 
+Reviewing Guidelines
+------------------------
+
+* Make sure the API docs are updated with the changes. If they are not, request
+  the contributor run `sphinx-apidoc` to update them.
```

Comment:
Friendly reminder that reStructuredText uses double back-ticks to pre-format the text, e.g.,

```rst
the contributor run ``sphinx-apidoc`` to update them.
```